locales-pending/landing-all.ftl

```diff
@@ -18,3 +18,7 @@ landing-all-value-prop-info-at-risk = What info could be at risk?
 landing-all-value-prop-info-at-risk-description = Data leaks are unfortunately part of our digital lives. Your passwords, contact details, financial information, and other personal info can be exposed, putting you at risk of identity theft.
 landing-all-value-prop-scanning-for-exposures-illustration-alt = Scanning for exposures progress bar
 landing-all-value-prop-leaked-password-illustration-alt = Leaked password example breach resolution step
+
+# Quote
```

Comment:
This is a stand-alone comment. What's the goal of this comment? It should probably be expanded a bit and tied to the following string (no empty line in-between). Same comment applies to the other string.